src/repr/src/namespaces.rs

```diff
@@ -13,22 +13,28 @@
 
 pub const MZ_TEMP_SCHEMA: &str = "mz_temp";
 pub const MZ_CATALOG_SCHEMA: &str = "mz_catalog";
+pub const MZ_CATALOG_UNSTABLE_SCHEMA: &str = "mz_catalog_unstable";
 pub const PG_CATALOG_SCHEMA: &str = "pg_catalog";
 pub const MZ_INTERNAL_SCHEMA: &str = "mz_internal";
+pub const MZ_INTROSPECTION_SCHEMA: &str = "mz_introspection";
 pub const INFORMATION_SCHEMA: &str = "information_schema";
 pub const MZ_UNSAFE_SCHEMA: &str = "mz_unsafe";
 
 pub const SYSTEM_SCHEMAS: &[&str] = &[
     MZ_CATALOG_SCHEMA,
+    MZ_CATALOG_UNSTABLE_SCHEMA,
     PG_CATALOG_SCHEMA,
     MZ_INTERNAL_SCHEMA,
+    MZ_INTROSPECTION_SCHEMA,
     INFORMATION_SCHEMA,
     MZ_UNSAFE_SCHEMA,
 ];
 
 // TODO: is `mz_unsafe` unstable too?
 pub const UNSTABLE_SCHEMAS: &[&str] = &[
+    MZ_CATALOG_UNSTABLE_SCHEMA,
     MZ_INTERNAL_SCHEMA,
+    MZ_INTROSPECTION_SCHEMA,
```

Comment:
Is this indeed unstable? I'm not certain what this schema will be used for.

Reply:
It will host the per-replica introspection sources, which curre try live in mz_internal. They are definitely unstable, with no plans to ever stabilize them.